runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -1475,14 +1533,14 @@ def TermDebugBalloonExpr(): string
   if v:beval_winid != sourcewin
     return ''
   endif
-  if !stopped
+  if stopped == false
```

Comment:
When using a Boolean variable, there is no need to it compare against true or false. The previous code is more readable.

Reply:
Done! I extended the process also in non-cited places that suffer of the same issue.